src/main/java/mate/academy/internetshop/dao/impl/UserDaoImpl.java

```diff
@@ -27,7 +27,7 @@ public User get(Long id) {
     @Override
     public User update(User toUpdate) {
         User user = get(toUpdate.getId());
-        user.setBucket(toUpdate.getBucket());
+        //user.setBucket(toUpdate.getBucket());
```

Comment:
do we need this line?

Reply:
I am going to uncomment it after bucket bound to user is available in webapp version